corehq/apps/export/models/new.py

```diff
@@ -915,6 +924,77 @@ def get_filters(self):
         return []
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = None
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        return cls(
+            domain=schema.domain,
+            tables=[
+                TableConfiguration(
+                    label="Messages",
+                    columns=[
+                        RowNumberColumn(),
+                        ExportColumn(
+                            label="Contact ID",
+                            item=ExportItem(
+                                path=[PathNode(name='couch_recipient')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Timestamp",
+                            item=ExportItem(
+                                path=[PathNode(name='date')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Username",
+                            item=ExportItem(
+                                path=[PathNode(name='couch_recipient')],
+                                transform=USERNAME_TRANSFORM,
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Phone Number",
+                            item=ExportItem(
+                                path=[PathNode(name='phone_number')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Direction",
+                            item=ExportItem(
+                                path=[PathNode(name='direction')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Message",
+                            item=ExportItem(
+                                path=[PathNode(name='text')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Type",
+                            item=ExportItem(
+                                path=[PathNode(name='workflow')],
+                                transform=WORKFLOW_TRANSFORM,
+                            ),
+                            selected=True,
+                        ),
+                    ],
+                    selected=True
```

Comment:
this tableconfiguration doesn't seem to have a `path` attribute. that should be required (didn't actually enforce at the db level, but should have). if it is set the `MAIN_TABLE` it will automatically be selected